.github/workflows/docker.yml

```diff
@@ -19,14 +20,27 @@ jobs:
     - name: Get commit hash
       id: get_commit
       run: echo "COMMIT=$(git rev-parse --short=7 HEAD)" >> $GITHUB_OUTPUT
+    - name: Get DAP draft
+      id: get_dap_draft
+      run: |
+        echo -n "DAP_DRAFT=" >> $GITHUB_OUTPUT
+        if [[ ${{ github.ref_name }} == "main" ]]; then
+          echo dap-draft-04 >> $GITHUB_OUTPUT
+        else
+          echo "${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
```

Comment:
FYI, this part is going to run with GITHUB_REF set to pull request merge branches, i.e. `refs/pull/198/merge`, and thus it'll tag some images (never to be pushed) as `divviup_ts_interop_client:merge`.

Reply:
Yes, I know. That seems silly, but we have to put _something_, and it's easier to use the nonsense string `merge` than to try to figure out what `dap-draft-0x` it should be.